modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectDefinitionLocalServiceImpl.java

```diff
@@ -661,6 +661,19 @@ public void deployObjectDefinition(ObjectDefinition objectDefinition) {
 		}
 	}
 
+	public void deployObjectDefinitions() {
```

Comment:
@pedro-oliveira446 override is missing, right?

Reply:
Just done!